### ISSUE TITLE
Enabling/disabling the thumbnail of widget

### ISSUE THREAD

**Issue by jacknorton** [itchio/itch.io#721]
A very simple feature request, add a parameter to turn the game image thumbnail on/off in the iframe widget :)
See this thread: https://itch.io/post/286081

**Comment by leafo** [comment]
Try again, I added a new breakpoint that hides thumbnail first before switching over to thumbnail only mode

**Comment by jacknorton** [comment]
Yes now works, though there's a side effect: the various purchase options don't work (if you have different prices for various items). Not sure why.

**Comment by jacknorton** [comment]
Did you have time to check this? or there are the widget sources somewhere so I could take a look maybe? thanks

**Comment by leafo** [comment]
Sorry, I didn't get a chance to look yet. It looks like we intentionally hide the file list because there isn't enough space to show them alongside. I'm wondering if we should just redirect to the full purchase page at this point instead of showing the files inside the widget. Any thoughts?

**Comment by jacknorton** [comment]
Well I actually changed my website design and now it's fine :) I think it's better how it's now since it's really easy for the users to purchase the game which is a good thing.
